napari/utils/key_bindings.py

```diff
@@ -182,6 +186,284 @@ def normalize_key_combo(key_combo):
 UNDEFINED = object()
 
 
+def call_with_context(function, context):
```

Comment:
the usage of `context` wasn't quite clear to me from reading this docstring and function, I eventually got it by figuring out what's supposed to call this function, and then the ActionManager docstring was kinda helpful, but it'd be nice to make that quicker to understand.